packages/tabs/src/TabTitle/TabTitle.styles.ts

```diff
@@ -104,10 +104,12 @@ export const listTitleFontSize: Record<BaseFontSize, string> = {
   [BaseFontSize.Body1]: css`
     font-size: ${typeScales.body1.fontSize}px;
     line-height: ${typeScales.body1.lineHeight}px;
+    height: 44px;
```

Comment:
use `spacing` tokens

Reply:
we don't have a matching spacing token and I don't think it makes sense to use spacing tokens for height. I instead extracted into constants